lib/flowViewport/sceneIndex/fvpPruneTexturesSceneIndex.cpp

```diff
@@ -60,16 +60,17 @@ _PruneTexturesFromMatNetwork(
 // static
 PruneTexturesSceneIndexRefPtr
 PruneTexturesSceneIndex::New(
-    const HdSceneIndexBaseRefPtr &inputSceneIndex)
+    const HdSceneIndexBaseRefPtr &inputSceneIndex,
+    bool pruneTextures)
 {    
     return TfCreateRefPtr(
-        new PruneTexturesSceneIndex(inputSceneIndex));
+        new PruneTexturesSceneIndex(inputSceneIndex, pruneTextures));
 }
 
 void
-PruneTexturesSceneIndex::MarkTexturesDirty(bool isTextured)
+PruneTexturesSceneIndex::MarkTexturesDirty(bool pruneTextures)
 {
-    _needsTexturesPruned = isTextured;
+    _pruneTextures = pruneTextures;
```

Comment:
Argument was called isTextured, but data member is the opposite logic, pruneTextures.  Made everything consistent.